clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/EnvIntegrationSpec.scala

```diff
@@ -50,16 +50,20 @@ abstract class EnvIntegrationSpec(env: String)
     rootPathForBucketInEnv(env, storageScopes)
 }
 
-/** The integration spec that runs against Clio in dev. */
-class DevEnvIntegrationSpec extends EnvIntegrationSpec("dev") with IntegrationSuite
+/** The integration specs that run against Clio in dev. */
+class DevEnvBasicSpec extends EnvIntegrationSpec("dev") with BasicTests
+class DevEnvUbamSpec extends EnvIntegrationSpec("dev") with UbamTests
+class DevEnvCramSpec extends EnvIntegrationSpec("dev") with WgsCramTests
+class DevEnvGvcfSpec extends EnvIntegrationSpec("dev") with GvcfTests
 
 /**
   * Integration spec checking that auth tokens properly refresh in the client.
   * Only meaningful if run against a deployed Clio with a running proxy.
   */
 class AuthIntegrationSpec extends EnvIntegrationSpec("dev") with AuthRefreshTests
 
-/** The integration spec that runs against Clio in staging. */
-class StagingEnvIntegrationSpec
-    extends EnvIntegrationSpec("staging")
-    with IntegrationSuite
+/** The integration specs that run against Clio in staging. */
+class StagingEnvBasicSpec extends EnvIntegrationSpec("staging") with BasicTests
```

Comment:
And this will break the clio-stage-RC job. It's not in job DSL yet so you can just update the job manually.

Reply:
Done.